src/assets/css/_sass/components/_home-made-local.scss

```diff
@@ -0,0 +1,38 @@
+.made-local-section {
```

Comment:
consider naming this `.made-local` and not including section. It lends itself more to component based architecture/bem

following classes can be renamed to `.made-local__header` to contain all their styling to the block

they could also be given a higher level class `.section__header`, `.section__title` for things that match other sections around the site.